### PR TITLE
cli: fix plugin install error, fix ci test case

### PR DESCRIPTION
**Pull request by FeelyChau**
The command `pipcook pipeline install /path/to/pipeline.json` finish with error, and CI did not check out this, because the plugins in the case are all installed before.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
         npm run test:pipeline -- text-bayes-classification
     - name: install plugins from specific pipeline
       run: |
-          ./packages/cli/dist/bin/pipcook pipeline install ./example/pipelines/text-bayes-classification.json
+          ./packages/cli/dist/bin/pipcook pipeline install ./example/pipelines/chinese-poem-creation.json
```
Comment (yorkie) [inline .github/workflows/build.yml:75]: What's the reason to change the test pipeline?

**Comment by yorkie** [comment]
> The command `pipcook pipeline install /path/to/pipeline.json` finish with error, and CI did not check out this, because the plugins in the case are all installed before.

May I ask you to point out where the current pipeline not checking?

File: packages/cli/src/service/pipeline.ts [hunk 2]
```diff
@@ -72,10 +72,8 @@ export async function installPackageFromConfig(config: any, opts: any): Promise<
   for (const plugin of constants.PLUGINS) {
     const packageName = config.plugins[plugin]?.package;
     if (typeof packageName === 'string') {
-      if (packageName[0] === '.') {
-        const pkg = await pluginInstall(packageName, opts);
-        config.plugins[plugin].package = pkg.name;
-      }
+      const pkg = await pluginInstall(packageName, opts);
```
Comment (yorkie) [inline packages/cli/src/service/pipeline.ts:75]: This might be better to put in another PR :)

File: packages/cli/src/service/pipeline.ts [hunk 3]
```diff
@@ -113,12 +111,12 @@ export async function install(filename: string, opts: any): Promise<void> {
     // check the installation
     logger.info('check plugin:');
     let isSuccess = true;
-    await constants.PLUGINS.map(async (plugin) => {
+    for (const plugin of constants.PLUGINS) {
```
Comment (yorkie) [inline packages/cli/src/service/pipeline.ts:114]: This will remove the concurrent running, do we have any specific reason for that?

Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:114]: The plugin project can't run concurrently, and the `await` can't take effect.

**Comment by FeelyChau** [comment]
> > The command `pipcook pipeline install /path/to/pipeline.json` finish with error, and CI did not check out this, because the plugins in the case are all installed before.
> 
> May I ask you to point out where the current pipeline not checking?

```shell
[start]: start install pipeline from ./example/pipelines/text-bayes-classification.json
[start]: installing plugins
[info]: start to create pipeline
[success]: pipeline is created: x8xo318f
[info]: check plugin:
[success]: pipeline installed successfully.
[success]: @pipcook/plugins-csv-data-collect installed.
[success]: @pipcook/plugins-csv-data-access installed.
[success]: @pipcook/plugins-bayesian-model-define installed.
[success]: @pipcook/plugins-bayesian-model-train installed.
[success]: @pipcook/plugins-bayesian-model-evaluate installed.
```
The log should be:
```shell
[success]: @pipcook/plugins-csv-data-collect installed.
[success]: @pipcook/plugins-csv-data-access installed.
[success]: @pipcook/plugins-bayesian-model-define installed.
[success]: @pipcook/plugins-bayesian-model-train installed.
[success]: @pipcook/plugins-bayesian-model-evaluate installed.
[success]: pipeline installed successfully.
```
If the pipeline plugins have not been install, it will throw error here.

File: packages/cli/src/service/pipeline.ts (hunk 3 above)
Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:114]: The original intent here was to install the plugin serial.

File: packages/cli/src/service/pipeline.ts (hunk 2 above)
Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:75]: This fixes an error that would not have been installed if the  name was not a local path.

File: packages/cli/src/service/pipeline.ts (hunk 3 above)
Comment (yorkie) [reply packages/cli/src/service/pipeline.ts:114]: Questions: a) What do you mean by the "plugin project" b) why does the `await` not taking effect?

File: .github/workflows/build.yml (hunk 1 above)
Comment (yorkie) [inline .github/workflows/build.yml:75]: So why do we change the pipeline to test here?

**Comment by yorkie** [comment]
@FeelyChau We have a label named `tests`, maybe that's more specific than `ci/cd` which is related to our workflows.

File: .github/workflows/build.yml (hunk 1 above)
Comment (FeelyChau) [reply .github/workflows/build.yml:75]: This makes a fresh installation with no plugins ware installed.

**Comment by FeelyChau** [comment]
> @FeelyChau We have a label named `tests`, maybe that's more specific than `ci/cd` which is related to our workflows.

OK, changed.

File: packages/cli/src/service/pipeline.ts (hunk 2 above)
Comment (yorkie) [reply packages/cli/src/service/pipeline.ts:75]: What error on that case?

File: packages/cli/src/service/pipeline.ts (hunk 3 above)
Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:114]: a) "plugin project" means the plugin nodejs project, we can't install package concurrently by mutil-process.
b) we can't await a map funtion.
test code:
```js
const arr = [1, 2, 3];
async function test() {
  await arr.map(async n => {
    setTimeout(() => console.log(n), 100);
  })
  console.log(4);
}

test();

```
output:
```shell
4
1
2
3
```

Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:114]: Question A may be not a problem after #455 , I'll test it.

File: packages/cli/src/service/pipeline.ts (hunk 2 above)
Comment (FeelyChau) [reply packages/cli/src/service/pipeline.ts:75]: `installPackageFromConfig ` should install plugins whatever the plugin name protocol(local path, url, git or npm package name).
But it accepts local path only.

Comment (yorkie) [reply packages/cli/src/service/pipeline.ts:75]: Ok, resolved.

**Review by yorkie — DISMISSED** [review]

**Comment by FeelyChau** [comment]
@yorkie The parallel installation has been restored. May I ask you to review again?

**Review by yorkie — APPROVED** [review]
LGTM
